specification/cognitiveservices/data-plane/Form/stable/v1.0/Form.json

```diff
@@ -0,0 +1,731 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "v1",
+        "title": "Cognitive Service Form Recognizer API V1 (Preview)",
+        "description": "Extracts information from forms and images into structured data based on a model created by a set of representative training forms."
+    },
+    "securityDefinitions": {
+        "apim_key": {
+            "type": "apiKey",
+            "name": "Ocp-Apim-Subscription-Key",
+            "in": "header"
+        }
+    },
+    "security": [
+        {
+            "apim_key": []
+        }
+    ],
+    "x-ms-parameterized-host": {
+        "hostTemplate": "{Endpoint}/form/v1.0",
```

Comment:
This means the final URL will be
`https://xxxxxxx/form/v1.0/formrecognizer/v1.0/custom/train`.
I don't think this is the case.

You probably want `{Endpoint}/formrecognizer/v1.0` in here, and remove `/formrecognizer/v1.0` prefix from all the operation paths.

Reply:
agreed, final URL will be: https://xxxxxxx/formrecognizer/v1.0/train